soc/arm64/nxp_imx/mimx9/pinctrl_soc.h

```diff
@@ -19,7 +19,6 @@ extern "C" {
 #define MCUX_IMX_DRIVE_OPEN_DRAIN_SHIFT IOMUXC1_SW_PAD_CTL_PAD_OD_SHIFT
 #define MCUX_IMX_BIAS_PULL_DOWN_SHIFT IOMUXC1_SW_PAD_CTL_PAD_PD_SHIFT
 #define MCUX_IMX_BIAS_PULL_UP_SHIFT IOMUXC1_SW_PAD_CTL_PAD_PU_SHIFT
-#define MCUX_IMX_BIAS_PULL_ENABLE_SHIFT IOMUXC1_SW_PAD_CTL_PAD_PE_SHIFT
```

Comment:
Just checking- is this field being dropped because it is not present on the hardware? I don't see a `PE` field in the IOMUXC registers for the iMX93, so I assume so.

Reply:
Correct. i.MX93 only uses PU and PD field.